RELEASE.md

```diff
@@ -0,0 +1,24 @@
+## How to release a new version of the project
+
+- Update the version in the `pom.xml` files in a new or existing branch using the following command:
+
+```shell
+mvn versions:set -DnewVersion=x.y.z
+```
+
+- Commit the changes and push the branch to the repository.
+- Create a pull request from the branch to the `main` branch.
+- Merge the pull request.
+- Create a new release in the repository with the same version as the one set in the `pom.xml` files.
+- The artifacts will be available in the GitHub Packages repository after the release is created.
+- Update the submodule `bankaxept-epayment-development-kit` and `epp-dev-kit.version` property in a new or existing
+   branch in the `bankaxept-epayment-platform` project using the following commands:
```

Comment:
This is not that crucial, but some instructions are specific to the main project and not necessarily releases in this repo. Since this is a public repository, should we really put this information outside? I'd consider adding parts of this to main repo docs instead.

Reply:
Sure, this section can either be moved or removed depending on how we decide to automate the release process in this repo.